public/css/style.css

```diff
@@ -94,18 +94,27 @@ nav input[type=search]:focus {
 }
 
 .sans-serif {
-  font-family: "Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif;
+  font-family: "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell,
+    "Helvetica Neue", sans-serif;
 }
 
-.input-field input:focus + label, .input-field textarea:focus + label{
-     color: var(--color-dark-blue) !important;
+.input-field input:focus + label,
+.input-field textarea:focus + label {
+  color: var(--color-dark-blue) !important;
 }
 
-.input-field input:focus, .input-field textarea:focus {
-     border-bottom: 1px solid var(--color-dark-blue) !important;
-     box-shadow: 0 1px 0 0 var(--color-dark-blue) !important;
+.input-field input:focus,
+.input-field textarea:focus {
+  border-bottom: 1px solid var(--color-dark-blue) !important;
+  box-shadow: 0 1px 0 0 var(--color-dark-blue) !important;
 }
 
 .divider {
-    background-color: var(--color-gold);
+  background-color: var(--color-gold);
+}
+
+#c {
+  background-color: rgb(113, 98, 122);
```

Comment:
I am ok with leaving this as is but if we wanted to remain consistent we would probably want to create a root var at the top of styles.css and make a var out of it. 

If this is just going to be used here I think it is ok but the moment we want to use it elsewhere that 'matches' we will want it in a variable. So that if we change it in one place it will change everywhere.